src/core/src/UrlAssetAccessor.cpp

```diff
@@ -0,0 +1,268 @@
+// Copyright(c) 2023 Timothy Moore
+// Copyright 2024 CesiumGS, Inc. and Contributors
+
+#include "cesium/omniverse/UrlAssetAccessor.h"
+
+#include <CesiumAsync/IAssetResponse.h>
+#include <CesiumUtility/Tracing.h>
+#include <omni/kit/IApp.h>
+
+#include <algorithm>
+#include <cstring>
+
+namespace cesium::omniverse {
+const auto CURL_BUFFERSIZE = 3145728L; // 3 MiB
```

Comment:
Do you know when the buffer gets freed? Is it when `curl_easy_reset` is called, or when the `CURL` object is destroyed? Hopefully not the former.

Reply:
Also I think this could be shrunk to 0.5 MiB given the avg tile sizes in CWT and Google.

Reply:
I believe curl_easy_reset() doesn't do much except to prepare for another request. For example, it doesn't tear down an existing connection. The buffer freeing should probably be verified, either by examining the code or with a debugger.